src/libraries/System.Formats.Tar/tests/Manual/ManualTests.cs

```diff
@@ -22,8 +22,9 @@ public static IEnumerable<object[]> WriteEntry_LongFileSize_TheoryData()
                 yield return new object[] { entryFormat, LegacyMaxFileSize, unseekableStream };
             }
 
-            // Pax supports unlimited size files.
+            // Pax and Gnu supports unlimited size files.
             yield return new object[] { TarEntryFormat.Pax, LegacyMaxFileSize + 1, unseekableStream };
+            yield return new object[] { TarEntryFormat.Gnu, LegacyMaxFileSize + 1, unseekableStream };
         }
     }
 
```

Comment:
Can't write below this line: Do you mind running the manual test in line 34? `WriteEntry_LongFileSize`. It's one of those tests that are difficult to run in CI.

Reply:
I ran the manual test in this project, and they all passed.